scripts/drone/events/pr.star

```diff
@@ -96,6 +96,7 @@ def pr_pipelines():
         test_backend(
             get_pr_trigger(
                 include_paths = [
+                    "Makefile",
```

Comment:
changing the makefile didn't trigger go tests

Reply:
Oh I see, you already addressed it.

Reply:
yeah, sorry, the way i worded that wasn't clear